lib/app.js

```diff
@@ -2,15 +2,18 @@ import React, { Component } from 'react'
 import PropTypes from 'prop-types'
```

Comment:
@HaNdTriX so what I was thinking. Why don't we just expose the context, and create a separate package that reads the context, that way we don't really have to add any code to Next, but provide a very powerful api 👌